src/core/Panel/__snapshots__/PanelExpansion.test.tsx.snap

```diff
@@ -274,8 +274,8 @@ exports[`calling render with the same component on the same container does not r
     </svg>
   </button>
   <div
-    aria-hidden="true"
     class="fi-panel-expansion_content c6 fi-panel c1 c2"
+    hidden=""
```

Comment:
@J-Kallunki is this ok? Seems to behave differently with test snapshots when compared to an actual browser.

Reply:
Also using css and visibility:hidden seems not to be working correctly. At least the hidden node remains in the accessibility tree for edge in with both use cases. When collapsing the group things seem to work better. BrowserStack was used for testing, so that might spice things up to some extent as well.

Reply:
Conditionally rendering the expansible content (not using hidden at all, but rather not rendering the whole component for the content) seems to work correctly with edge and accessibility tree. This however is not probably the best approach to achieve the functionality.

Currently used aria-hidden attribute is correctly reflected to dom as well, but does not reflect to Edge's accessibility tree.

Reply:
Tested this in Edge on Windows 10 and it behaved as above. hidden="" prop remained as well as the accessibility tree element, even though the element behaved correctly in the view.

Reply:
To sum things up:
- `hidden` attribute does not seem to behave as expected in Edge/IE
- `visibility: hidden` has also no effect on the accessibility tree in Edge/IE
- conditional rendering of the content seems to work as well as using aria-hidden.

So any combination of these is possible, but Edge and IE require either using aria-hidden in combination with visual hiding technique or conditionally hiding the actual content. If content is conditionally displayed, it might make sense to wrap the content and apply visual hiding there so that animations and other visual representations work as expected.

Waiting for comment from @J-Kallunki before progressing with this.

Reply:
Using key to force render of expansible content seems to resolve the issue with hidden attribute remaining in dom. So using conditional hidden prop / attribute with aria-hidden as fallback seems to be working with all browsers.